TTS/utils/generic_utils.py

```diff
@@ -58,28 +58,15 @@ def get_commit_hash():
     return commit
 
 
-def create_experiment_folder(root_path, model_name):
```

Comment:
I guess this is what you mean above but not sure if it solves the case I explained above

Reply:
Yeah! I replied above 🙂